tests/test_pkg_integrity.py

```diff
@@ -94,7 +91,7 @@ def _mock_fetch_shasum(url):
     @mock.patch('pkg_integrity.GnomeOrgVerifier.fetch_shasum', _mock_fetch_shasum)
     def test_gnome_org(self):
         result = self.run_test_for_domain(pkg_integrity.GnomeOrgVerifier, GNOME_SHA256_PKG)
-        self.assertTrue(result)        
+        self.assertTrue(result)
 
 
 @mock.patch('pkg_integrity.download_file', mock_download_file)
```

Comment:
Is this `@mock.patch` (and others) still needed, since you are removing the download_file calls in tests?

Reply:
Yes, the download_file function is called by most of the tests under the pkg_integrity.check() function.

Reply:
Oh, right, thanks for clarifying that.